source/applications/psyllid.cc

```diff
@@ -2,19 +2,22 @@
  * psyllid.cc
  *
  *  Created on: Feb 1, 2016
- *      Author: nsoblath
+ *      Author: N.S. Oblath
  */
 
-#include "psyllid_constants.hh"
+#include "daq_control.hh"
 #include "psyllid_error.hh"
 #include "psyllid_version.hh"
-#include "run_server.hh"
+
+#include "conductor.hh"
+#include "sandfly_error.hh"
 #include "server_config.hh"
 
 #include "application.hh"
 #include "logger.hh"
 
 using namespace psyllid;
+using namespace sandfly;
```

Comment:
sandfly_error.hh and psyllid_error.hh are both included.
1. What's the difference? They look almost similar. But one apparently has the deprecated equivalent of "noexcept".
2. Both namespaces are used as well. What would the class type `error` redirect to, `sandfly_error::error` or `psyllid_error::error` ? Will it be the correct one? 

My guess is that psyllid_error has been replaced by sandfly_error without removing it?

Reply:
The two error classes exist to (1) provide the useful streaming interface for setting the `what()` message (I know that `psyllid::error` really should use the same base class as `sandfly::error`) and (2) provide some identification as to the source of the error broadly, whether it comes from the framework (i.e. sandfly) or the DAQ implementation (i.e. psyllid).

I think that, in principle, if one were to bring both error classes into the global namespace with a `using` directive, the compiler should complain about the ambiguity.  But that's why we have namespaces, and hopefully no one is using `using namespace psyllid` or `using namespace sandfly`, and especially not together.

Reply:
With my comment I wanted to point out that this source file indeed does have both `using namespace psyllid` and `using namespace sandfly` and using the class `error()` just like that would result in ambiguity.
But I have noticed only after I made that comment that this file only uses `psyllid::error` and `sandfly::error` with their full names. I guess it is okay to do that actually. When I first saw both errors included I just did not think it was intentional.